src/main/java/com/segmentreader/dataformats/Convertor.java

```diff
@@ -0,0 +1,26 @@
+package com.segmentreader.dataformats;
+
+import java.io.IOException;
+import java.util.LinkedList;
+
+import org.apache.hadoop.io.Text;
+
+import com.segmentreader.mapreduce.UserModCommand;
+
+public class Convertor {
```

Comment:
Lets create interface Convertor and rename it properly. Also this class should have name like ConvertorImpl